test-suite/src/blend.rs

```diff
@@ -159,6 +160,41 @@ test_case!(blend, async move {
             EvaluateError::ValueNotFound("noname".to_owned()).into(),
             "SELECT noname FROM BlendUser",
         ),
+        (
+            EvaluateError::ValueNotFound("noname".to_owned()).into(),
+            "SELECT * FROM BlendUser WHERE noname = 1;",
+        ),
+        (
+            EvaluateError::NestedSelectRowNotFound.into(),
+            "SELECT * FROM BlendUser WHERE id = (SELECT id FROM BlendUser WHERE id = 9);",
+        ),
+        (
+            FetchError::TableNotFound("Nothing".to_owned()).into(),
+            "SELECT * FROM Nothing;",
+        ),
+        (
+            TranslateError::TooManyTables.into(),
+            "SELECT * FROM BlendUser, BlendItem",
+        ),
+        (
+            TranslateError::UnsupportedStatement("TRUNCATE TABLE BlendUser".to_owned()).into(),
+            "TRUNCATE TABLE BlendUser;",
+        ),
```

Comment:
`blend.rs` is for testing.. blend module or in SQL it is commonly known as projection.
These does not look related to blend.
It would be good to find another places.

e.g.
This.. looks good to be located in `join` because it shows that implicit inner join is not supported.
```rust
            TranslateError::TooManyTables.into(),
            "SELECT * FROM BlendUser, BlendItem",
```
This would be good to exist in `nested_select`
```rust
        (
            EvaluateError::NestedSelectRowNotFound.into(),
            "SELECT * FROM BlendUser WHERE id = (SELECT id FROM BlendUser WHERE id = 9);",
        ),
```